client/blocks/reader-post-card/gallery.jsx

```diff
@@ -14,7 +14,7 @@ import cssSafeUrl from 'lib/css-safe-url';
 import { isFeaturedImageInContent } from 'lib/post-normalizer/utils';
 import ReaderPostCardExcerpt from './excerpt';
 
-const GALLERY_ITEM_THUMBNAIL_WIDTH = 420;
+const READER_CONTENT_WIDTH = 800;
```

Comment:
Should this be 800 or 720? 
I think it would be better if we reused (and/or updated) all of the constants from `client/state/reader/posts/normalization-rules.js`